### DISCUSSION TITLE
首页推送的直播 右键菜单快速设置过滤

### DISCUSSION THREAD

**Discussion by sharkucn** [ywmoyue/biliuwp-lite#908] [bug]
### Bug 描述

右键单击推送在首页的直播，打开的菜单为空，点击没有反应。
视频的右键菜单正常。

### 操作步骤

```plain text
1. 打开应用
2. 在推荐页任意找到一个直播
3. 右键单击可见
```

### 期望结果

正常显示“不感兴趣”按键

### 截图

直播的右键菜单有问题![bugged](https://github.com/user-attachments/assets/142ed607-981c-42d3-8d4e-eb14978e8082)

视频的右键菜单显示与功能均正常![normal](https://github.com/user-attachments/assets/25dd2ad0-a58e-4a35-ae51-983f125cc96f)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 11 Dev (Build 27729.1000) (Canary Channel)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 备注

应用版本 4.7.6.0

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
改成需求吧，后续版本给这里加个快速设置过滤器的按钮

**Reply by ywmoyue** [reply]
https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.6.703 @sharkucn 试试

**Reply by sharkucn** [reply]
标签页的问题修正了，快捷过滤也能正常设置，感谢您的开发
